x-pack/plugins/lens/public/indexpattern_datasource/operations/layer_helpers.ts

```diff
@@ -30,6 +32,11 @@ import { generateId } from '../../id_generator';
 import { ReferenceBasedIndexPatternColumn } from './definitions/column_types';
 import { FormulaIndexPatternColumn, regenerateLayerFromAst } from './definitions/formula';
 
+interface ColumnFilters {
```

Comment:
The whole PR is using "filters" as a name of the extra params passed around like this. While they are just filters for now, it's more about additional params - could we rename this to `incompleteParams`, `ColumnParams` and so on?